data/campaigns/Northern_Rebirth/scenarios/05a_01_The_Pursuit.cfg

```diff
@@ -2124,12 +2124,12 @@
 
         [message]
             speaker=Abhai
-            message= _ "(<i>pales if possible</i>) Malifor...! Finally someone here in force to deal with that menace!"
+            message= _ "Malifor...! Finally someone here in force to deal with that menace!"
         [/message]
 
         [message]
             speaker=Tallin
-            message= _ "Can you tell us where he is? And, if I may ask, who are — or rather, were — you?"
+            message= _ "<i>Did this spirit get even paler?</i> Can you tell us where he is? And, if I may ask, who are — or rather, were — you?"
```

Comment:
I would separate the two questions, not putting them on consecutive lines of the same message. One doesn't ask two questions in immediate sequence without a pause to ponder the first one.

Reply:
Also, "pale spirit" is a bit confusing, it's alluding to some sort of symbolic/moral aspect rather than the physicality, i.e. the wraith's spirit is paler as opposed to the wraith being paler. I realize that's not what you meant, but that's what it sounds like.

Reply:
Two questions in one line is odd, but I think it fits the flow here - Tallin first asks the question related what he's currently trying to do, and then realises that he needs to build rapport with the new-found source of information first.

Reply:
Yes, but I'd let him have a piece of dialog to himself, then for the user to press next/ok, then for him to talk to somebody else. So there's pause.

Reply:
> I would separate the two questions, not putting them on consecutive lines of the same message. One doesn't ask two questions in immediate sequence without a pause to ponder the first one.

One of them is a thought, to himself, and that is marked by the italic font. In game, you would not read them back to back with the same voice, you would unconsciously separate them, because one question is in italics and the next is not. 

> Also, "pale spirit" is a bit confusing, as if it was alluding to some sort of symbolic/moral aspect rather than the physicality.

I think the context clears that up, as you were talking to a ghost, and there would not be a subject for that moral reference. (e.g. Tallin's units, himself, Abhai... would not be referenced, not even by context for that moral aspect to make sense). To clear things up, for that to happen, in my opinion, the subject should be there in a sentence similar to:

`Did this wraiths' spirit get even paler?`
`Did his spirit get even paler?`
`Did my men's spirit get even paler?`

And that does not happen.

Reply:
Including the unspoken thought, there's three questions in this line.

Reply:
> Including the unspoken thought, there's three questions in this line.

Uhhhh, true, but that was an issue before aswell then, as the two spoken questions have not been changed. I'd say that would be something to be considered in another issue.

Reply:
>  as you were talking to a ghost,

The adjective "pale" does not apply to the noun "ghost", IMHO.

As for @stevecotton 's comment - before, it was 2 questions, now it's 3. 

As for whether to file another issue - if we agree here on some text, we can always file another issue, that's not the point...